deploy/src/main/resources/quick-start/README.md

```diff
@@ -0,0 +1,214 @@
+
+	The `autoscale.scalingtarget` should be the RabbitMQ queue name to monitor. Set `autoscale.maxinstances` and `autoscale.mininstances` as your quality of service and resources dictate. The `autoscale.backoff` is the number of intervals to skip monitoring after a scale up or down command is issued. This prevents unusual values being considered when the system is in an unstable state.
+
+	Deploy/redeploy the services and the autoscale container. After one or two minutes the autoscale container should find the services and start monitoring.
+
+	##### Example
+
+	The Job Service microservices already have these labels specified. You can see the labels set under some of the components, for example with worker-langdetect.
+
+	```
+    	labels:
+      	- autoscale.metric=rabbitmq
+      	- autoscale.scalingtarget=languageidentification-in
+      	- autoscale.scalingprofile=default
+      	- autoscale.maxinstances=4
+      	- autoscale.mininstances=1
+      	- autoscale.interval=30
+      	- autoscale.backoff=10
+	```
+
+
+4. **Deploy the services**
+ 
+ 	Navigate to the folder where you have downloaded the files to and issue the following command to deploy the services through Docker Stack. Each service provided in the compose file will be deployed to the same stack. The Autoscaler needs to be on the same stack in order to monitor other services. In the command below the stack name is `autoscalerdemo`
+
+	<table>
+      <tr>
+        <td><b>Docker Stack</b></td>
+        <td>docker stack deploy --compose-file=docker-compose.yml autoscalerdemo</td>
+      </tr>
+    </table>
+
+	##### Example
+
+	Running this command with the docker-compose.yml file supplied will bring up the Job Service as well as the Autoscaler Service.
+
+## Testing the Autoscaler
+
+#### Healthcheck
+
+The [autoscale-core](https://github.com/Autoscaler/autoscaler/blob/develop/autoscale-core) application inherently exposes standard and module-specific health checks. If you expose the admin port (default 8081) then this can be accessed via HTTP to examine metrics and health checks. The health check REST call will return HTTP 500 if any health check fails. For details on the health checks for specific components, examine the module documentation.
+
+#### File Testing
+
+With the Job Service deployed you can test if the Autoscaler is running correctly by supplying some files in the input directory and observing the services scaling up and down.
+
+The Input directory can be configured through an environment variable, otherwise it will be set to its default value.
+
+`- ${JOB_SERVICE_DEMO_INPUT_DIR:-./input-files}:/mnt/caf-worker-input-dir:ro`
+
+For further information on what type of files to supply to the Job Service, refer to the [Worker-Globfilter repository](https://github.com/JobService/worker-globfilter).
```

Comment:
Update to reference the actual README, and use a released version something like this ( not on develop ).

https://github.com/JobService/worker-globfilter/blob/develop/README.md